e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -4,8 +4,7 @@ exports[`CLI help > should print help with argument ---help 1`] = `
 "[1mCode PushUp CLI[22m
 
 Commands:
-  code-pushup               Shortcut for running collect followed by upload
-                                                                       [default]
+  code-pushup               Run Plugins and collect results            [default]
   code-pushup autorun       Shortcut for running collect followed by upload
```

Comment:
If we are to make collect a default command, wouldn't it make sense to rename `autorun` (implying default behaviour) to something else like "full run"?

Reply:
Maybe it's better not to have a default command? Lighthouse CI has `autorun`, but no default.

Also worth noting that if the user doesn't add any `upload` config, then `autorun` is equivalent to `collect` anyway.

Reply:
I would have collect as default as it is the minimum text in the terminal to run it :D

Reply:
Sure, but then why not keep `autorun` as default? Since we skip upload when the config is missing, then I'd say we have a pretty good flow:

- for users without portal, `npx code-pushup` runs `collect`
- for users with portal, `npx code-pushup` runs `autorun`

Isn't this what we want? If you change the default, then portal users have to type more. For CLI-only users, there's no difference.

Reply:
So is the error of requiring the portal client fixed now and I would just see the a warning message for missing uplaod?

Reply:
Yes, that error should have been fixed by https://github.com/code-pushup/portal/issues/244